discovery-provider/src/queries/get_health.py

```diff
@@ -340,9 +357,12 @@ def get_health(args: GetHealthArgs, use_redis_cache: bool = True) -> Tuple[Dict,
         "latest_indexed_block_num": latest_indexed_block_num,
         "final_poa_block": final_poa_block,
         "transactions_history_backfill": {
-            "index_user_backfilling_complete": is_index_user_bank_backfill_complete,
+            "user_bank_backfilling_complete": is_index_user_bank_backfill_complete,
```

Comment:
Lets stick all of these under `verbose` since they're not going to be needed by the client and are just used by us to see status